RustFxA/FirefoxAccountSignInViewController.swift

```diff
@@ -96,36 +96,23 @@ class FirefoxAccountSignInViewController: UIViewController, Themeable {
         }
     }
 
-    private lazy var scanButton: ResizableButton = .build { button in
-        button.layer.cornerRadius = UX.buttonCornerRadius
+    private lazy var scanButton: PrimaryRoundedButton = .build { button in
+        let viewModel = PrimaryRoundedButtonViewModel(
+            title: .FxASignin_QRScanSignin,
+            a11yIdentifier: AccessibilityIdentifiers.Settings.FirefoxAccount.qrButton
+        )
+        button.configure(viewModel: viewModel)
         button.setImage(self.signinSyncQRImage?.tinted(withColor: .white), for: .highlighted)
-        button.setTitle(.FxASignin_QRScanSignin, for: .normal)
-        button.accessibilityIdentifier = AccessibilityIdentifiers.Settings.FirefoxAccount.qrButton
-        button.titleLabel?.font = DefaultDynamicFontHelper.preferredBoldFont(
-            withTextStyle: .callout,
-            size: UX.buttonFontSize)
-
-        let contentPadding = UIEdgeInsets(top: UX.buttonVerticalInset,
-                                          left: UX.buttonHorizontalInset,
-                                          bottom: UX.buttonVerticalInset,
-                                          right: UX.buttonHorizontalInset)
-        button.setInsets(forContentPadding: contentPadding, imageTitlePadding: UX.buttonHorizontalInset)
```

Comment:
Looks like we need to add on the configure method a way to pass in this `imageTitlePadding` so we can keep the same spacing between the after and before. We can maybe add an optional param in `PrimaryRoundedButtonViewModel` and use it in `PrimaryRoundedButton`. We'll also need to make the adjustments to that view model usage probably in the [SampleApplication](https://mozilla-mobile.github.io/firefox-ios/documentation/componentlibrary/gettingstarted/) (a standalone application used to showcase the different components we have in the app).

Reply:
I've pushed an update, it should bring back the spacing between the image and the title

<details>
  <summary>Before / After, Light mode</summary>

![before-after-light-spacing](https://github.com/mozilla-mobile/firefox-ios/assets/26223450/119f150a-86f0-44a6-a96c-5273670d5c17)

</details>

<details>
  <summary>Before / After, Dark mode</summary>

![before-after-dark-spacing](https://github.com/mozilla-mobile/firefox-ios/assets/26223450/2ff33b40-c365-4ec8-8c3b-fcaa3169722f)

</details>